buildlib/azure-pipelines-pr.yml

```diff
@@ -163,6 +166,13 @@ stages:
               make -s -j `nproc`
             displayName: Build for $(CONTAINER)
 
+          - bash: |
+              set -eE
+              cd build
+              ./src/tools/info/ucx_info -e -u t
```

Comment:
i'd suggest to save the output to file (|&tee info.txt) and grep for any warn/error